src/GraphQl/Resolver/Stage/SecurityPostDenormalizeStage.php

```diff
@@ -44,6 +44,11 @@ public function __invoke(string $resourceClass, string $operationName, array $co
         $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
 
         $isGranted = $resourceMetadata->getGraphqlAttribute($operationName, 'security_post_denormalize', null, true);
+
+        if (null === $this->resourceAccessChecker && $isGranted) {
+            throw new \LogicException('Cannot check security expression when SecurityBundle is not installed. Try running "composer require symfony/security-bundle".');
+        }
+
```

Comment:
I think we have an issue if `null === $this->resourceAccessChecker` and `$isGranted` is false (see the code below).

Reply:
Fixed with @teohhanhui suggestions.